styles/XH.scss

```diff
@@ -51,7 +51,7 @@ body.xh-app {
     margin: var(--xh-pad-px) 0;
   }
 
-  :focus:not(.bp3-input),
+  :focus:not(.bp3-input, .bp3-slider-handle),
   .bp3-control input:focus ~ .bp3-control-indicator {
```

Comment:
Not sure why this class sequence does not override the blue print outline for radios, checkboxes, and switches.
The same class sequence with outline:0 in the new kit/blueprint/styles.scss change does.

Reply:
Were you testing in dark theme? It could be the inclusion of `.bp3-dark` in kit/blueprint/styles.scss